test/e2e/tests/desk/inspectors.spec.ts

```diff
@@ -0,0 +1,60 @@
+import {test, expect} from '@playwright/test'
+
+test.describe('sanity/desk: document inspectors', () => {
+  test('open and close custom inspector', async ({page}) => {
+    await page.goto('/test/content/input-debug;inspectorsTest;inspectors-test')
```

Comment:
I suggest we create a unique document when running these tests, instead of relying on already existing documents. We test review changes (the document must be a draft) and validation (the document needs validation errors) - and if someone edits or publishes the documents currently used, then a test might fail. 

```ts
import {uuid} from '@sanity/uuid'
import {testSanityClient} from '../../helpers'

test('open "Review changes" inspector', async ({page}) => {
  const id = uuid()
  await testSanityClient.create({
    _type: 'stringsTest',
    _id: id,
  })

  await page.goto(`/test/content/input-standard;stringsTest;${id}`)

  // ...

  await testSanityClient.delete(id)
})

```



We have a dedicated folder (`/schema/ci`) in test-studio, I think it would be good to move the tests to that folder!